<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OSGEarthClass</class>
 <widget class="QMainWindow" name="OSGEarthClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OSGEarth</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>几何图形</string>
    </property>
    <addaction name="actionLine"/>
    <addaction name="actionPolygon"/>
    <addaction name="actionCircle"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>测量分析</string>
    </property>
    <addaction name="actionDistance"/>
    <addaction name="actionHeight"/>
    <addaction name="actionArea"/>
    <addaction name="actionWater"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>通视分析</string>
    </property>
    <addaction name="actionLineAns"/>
    <addaction name="actionAreaAns"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Avatar</string>
    </property>
    <addaction name="actionAvatar"/>
    <addaction name="actionOverlay"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="actionGride"/>
    <addaction name="actionMap"/>
    <addaction name="actionSwitchMap"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>线</string>
   </property>
  </action>
  <action name="actionPolygon">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>多边形</string>
   </property>
  </action>
  <action name="actionCircle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>圆</string>
   </property>
  </action>
  <action name="actionHeight">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>高度测量</string>
   </property>
  </action>
  <action name="actionDistance">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>长度测量</string>
   </property>
  </action>
  <action name="actionArea">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>面积测量</string>
   </property>
  </action>
  <action name="actionWater">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>水淹分析</string>
   </property>
  </action>
  <action name="actionLineAns">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>两点通视</string>
   </property>
  </action>
  <action name="actionAreaAns">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>区域通视</string>
   </property>
  </action>
  <action name="actionAvatar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>控制</string>
   </property>
  </action>
  <action name="actionOverlay">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Overlay</string>
   </property>
  </action>
  <action name="actionGride">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>网格</string>
   </property>
  </action>
  <action name="actionMap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>小地图</string>
   </property>
  </action>
  <action name="actionSwitchMap">
   <property name="text">
    <string>切换地图</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="osgearth.qrc"/>
 </resources>
 <connections/>
</ui>
